README.md

```diff
@@ -101,6 +101,8 @@ We encourage the usage of common, machine readable standards such as OpenAPI, Gr
 - **Productivity**: Teams work with a contract in mind and are able to work in parallel
 - **Transparency**: Commitment and agreement on what is expected from an engineering standpoint
 
+Our current implementation of this principle can be visualized on this <a href="./docs/apifirst-flow.svg" target="_blank">API-first RFC Flow</a>.
```

Comment:
I wouldn't call it "our implementation" necessarily, since it's not really shared across all teams and not enforced in any way.

More like "a reference implementation" :)